src/tga/decoder.rs

```diff
@@ -505,12 +505,12 @@ impl<'a, R: 'a + Read + Seek> ImageDecoder<'a> for TGADecoder<R> {
         (self.width as u64, self.height as u64)
     }
 
-    fn colortype(&self) -> ColorType {
+    fn color_type(&self) -> ColorType {
         self.color_type
     }
 
     fn scanline_bytes(&self) -> u64 {
-        self.row_bytes()
+        self.color_type.bytes_per_pixel() as u64 * self.width as u64
```

Comment:
Should have some justification of cast/overflow behaviour. I believe it can not if `bytes_per_pixel` is `u8`.